### PR TITLE
172877789 te mc wrapper

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR includes minor updates to the custom message callback which is needed to send messages from the TE plugin to the MC Library Interactive.  Also includes updates to the sample data to make testing a wrapped MC question easier.

### CODE REVIEW

File: src/components/activity-page/managed-interactive/iframe-runtime.tsx [hunk 1]
```diff
@@ -246,7 +246,8 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
       // Re-init interactive, this time using a new mode (report or runtime).
       phoneRef.current = new iframePhone.ParentEndpoint(iframeRef.current, initInteractive);
       setSendCustomMessage((message: ICustomMessage) => {
-        phoneRef.current?.post(message.type, message.content);
+        // TODO: structure message type and content correctly
+        phoneRef.current?.post("customMessage", message);
```
Comment (mklewandowski) [inline src/components/activity-page/managed-interactive/iframe-runtime.tsx:250]: My understanding was that we post "customMessage" and the message itself contains a `type` and optional `content`.  In the TE case that we've been working on, the `type` on the `message` would specify showing/hiding overlays and the `content` could be empty.  Do I have this correct or am I not understanding the message structure?

**Review by kswenson — APPROVED** [review]
👍 LGTM

File: src/components/activity-page/managed-interactive/iframe-runtime.tsx (hunk 1 above)
Comment (scytacki) [reply src/components/activity-page/managed-interactive/iframe-runtime.tsx:250]: Yes that is how I thought it should work.

**Review by scytacki — CHANGES_REQUESTED** [review]
This looks good to me.
I'd remove the `TODO` since I don't think there is anything needed there.

My only concern is that it seems there is no test that is actually triggering the `post("customMessage", message)`

It looks like that iframe-runtime.tsx file is covered some by cypress and some by jest.
I don't know the activity player and its test well enough to know if this is something easy to cover or if it would be hard.

**Comment by mklewandowski** [comment]
> This looks good to me.
> I'd remove the `TODO` since I don't think there is anything needed there.
> 
> My only concern is that it seems there is no test that is actually triggering the `post("customMessage", message)`
> 
> It looks like that iframe-runtime.tsx file is covered some by cypress and some by jest.
> I don't know the activity player and its test well enough to know if this is something easy to cover or if it would be hard.

I removed the TODO comment - I put that in only to make sure we looked at it before merging.  Not sure on the test.  I would have assumed any needed tests were in place in past PRs.  Not sure exactly how we would test that.  Will require more thought.

File: src/components/activity-page/managed-interactive/iframe-runtime.tsx (hunk 1 above)
Comment (mklewandowski) [reply src/components/activity-page/managed-interactive/iframe-runtime.tsx:249]: Thanks.  I removed the now obsolete comment as well.  I would vote that we merge this so I can move on to other work.

**Review by scytacki — APPROVED** [review]
Sounds good.

**Pull request merged**